src/app.ts

```diff
@@ -13,14 +13,85 @@ interface TemperatureSummary {
   average: number
 }
 
+interface daySumary {
+  time: string
+  temperature: number
+  city: string
+  arrivalNumber: number
+}
+
+const listDaySumary: daySumary[][] = [];
+let sumaryOfDay: daySumary[] = [];
+let tempReading: TemperatureReading;
+let order: number;
+
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  tempReading = readings[0];
+  order = 1;
```

Comment:
Same here, this variable was used to enumerate the order but if you are using an array you can use some other methods